hnn_core/tests/test_optimization.py

```diff
@@ -136,3 +136,91 @@ def test_optimize_evoked():
 
     # static drive params should remain constant
     assert drive_static_params_opt == drive_static_params_orig
+
+
+def test_optimize_evoked_selectedDrive():
+    """Test running optimzing a selected drive while parameters of other drives
+    stay the same in a reduced network."""
```

Comment:
nitpick but [python style conventions](https://peps.python.org/pep-0257/#one-line-docstrings) dictate that this should be one line.